data/sql/updates/pending_db_world/rev_1669870365832502100.sql

```diff
@@ -0,0 +1,5 @@
+--
+-- Improve Mazthoril Recipe Drop Rates
+UPDATE `creature_loot_template` SET `Chance`=6 WHERE  `Entry`=7437 AND `Item`=14493 AND `Reference`=0 AND `GroupId`=0;
+UPDATE `creature_loot_template` SET `Chance`=2 WHERE  `Entry`=7437 AND `Item`=16054 AND `Reference`=0 AND `GroupId`=0;
+UPDATE `creature_loot_template` SET `Chance`=10 WHERE  `Entry`=7437 AND `Item`=13497 AND `Reference`=0 AND `GroupId`=0;
```

Comment:
```suggestion
UPDATE `creature_loot_template` SET `Chance`=6 WHERE `Entry`=7437 AND `Item`=14493 AND `Reference`=0 AND `GroupId`=0;
UPDATE `creature_loot_template` SET `Chance`=2 WHERE `Entry`=7437 AND `Item`=16054 AND `Reference`=0 AND `GroupId`=0;
UPDATE `creature_loot_template` SET `Chance`=10 WHERE `Entry`=7437 AND `Item`=13497 AND `Reference`=0 AND `GroupId`=0;
```
Spacing fix. ^_^

Reply:
word I'll try to look out for that, it was done by HeidiSQL automatically, I'll see if I can mess with the default outputs tho for the future.